ci/jenkins/test.sh

```diff
@@ -327,12 +327,15 @@ function prepare_env {
 function revert_snapshot_windows {
     WIN_NAME=$1
     echo "==== Reverting Windows VM ${WIN_NAME} ====="
-    if [[ $WIN_NAME == *"jumper"* ]]; then
-        govc snapshot.revert -vm ${WIN_NAME} win-initial
+    if [[ ${TESTCASE} =~ "containerd-ovs" ]]; then
+        if [[ $WIN_NAME == *"jumper"* ]]; then
```

Comment:
Has this image node name changed?

Reply:
i have made the change but for a-ms-0019 testbed the  node  a-ms-0019-jumper-win-0 also needs to be reverted to win-initial , so we will have to keep this condition right?

Reply:
@XinShuYang Can you clarify this?